lib/src/css_mirroring/util_classes.dart

```diff
@@ -0,0 +1,69 @@
+part of scissors.src.css_mirroring.bidi_css_generator;
```

Comment:
Don't use parts so eagerly: it's simpler to keep these as libraries, then import them, handle their imports separately, etc. (imports play a role in readability, especially with the show / as syntax)

Also, means you won't need to make all these types private (dropping polluting _ throughout)